docs/cookbook/recipe_delete_field_group.rst

```diff
@@ -3,7 +3,7 @@ Deleting a Group of Fields from an Admin
 
 In some cases, when you extend existing Admins, you might want to delete
 fields from the admin, or make them not show. You could delete every
-field by hand, using the ``FormMapper``s ``remove`` method.
+field by hand, using the ``FormMapper`` ``remove`` method.
```

Comment:
I think this is supposed to be [Saxon genitive](https://multimedia-english.com/grammar/saxon-genitive-s-possession-13), but it misses an apostrophe.

I can think of 3 fixes:

- `using ``FormMapper::remove()`` `
- `using the ``FormMapper::remove()`` method`
- `using the ``remove`` method of the ``FormMapper`` class.

Pick whichever you prefer.

Reply:
I choose the third :+1: